### DISCUSSION TITLE
Dependabot daily job fail - "Update check cancelled' error

### DISCUSSION THREAD

**Discussion by DanyC97** [community/community#58844] [Enterprise] [answered]
### Select Topic Area

Bug

### Body

hi folks,

In my org, on all my private repos (~ 50 repos) i setup a basic dependabot.yaml which checks daily for `package-ecosystem: terraform` OR `package-ecosystem: github-actions`.

If i trigger the jobs manually they all run successful and i can see the logs. However getting triggered daily is a hit and miss, mostly  miss with the below error.

Since i have no clue how to troubleshoot, anyone from dependabot team can chime in please? 

There are no logs, speaking with GH org admin, no clue where to dive in (not is a hosted org in github.com not GH enterprise server), how do you go and troubleshoot?

![image](https://github.com/community/community/assets/7383422/d285aae1-8374-40ec-ab6c-706fcc24610a)

**Comment by DanyC97** [comment]
Hi folks, 

any chance someone from the dev eng team might provide some insights as to where to dig next ?

**Comment by jana4u** [comment]
Having the exact same issue. Any info appreciated.

**Comment by Nishnha** [comment]
Hi 👋🏾 I'm an engineer on the Dependabot Updates team and I think I may have tracked down a cause for this error.

I think if you have your Dependabot update configs scheduled for the same time across a lot of repos our backend sees it as spammy and ends up rate limiting the the entire org.

I would recommend either (1) configuring your Dependabot version updates to run at different times, or (2) forego setting a time in your Dependabot configs altogether so that Dependabot can run updates across the scheduled day.

I will look into how we can mitigate this issue on our end, and please let me know if this helps!

**Reply by Nishnha** [reply, marked as answer]
Update on this: we had a support engineer confirm that changing the update schedule has no affect and is still seeing "update check cancelled" errors.

We will be automatically retrying some of these update check jobs on our backend instead of cancelling the job when we encounter this error, so you should see more successful runs.

**Comment by DanyC97** [comment]
this issue has been solved back in July

**Reply by spxnstevens** [reply]
Hi, any update to how it was solved?  I have this on two repo's and am flying blind on errors etc to check, just that "Update check cancelled" error.  Thanks :)

**Comment by matt-mckinstry-cloudm** [comment]
This is happening to us. We have 5 repos set to interval: monthly. Only 2 of them succeed

**Comment by Zippy1999** [comment]
Our team has the same problem. This started happening at the beginning of December 2024 and our jobs are still not running automatically. Is there any update on this please?

**Reply by marcreichman-pfi** [reply]
We are also seeing all of our jobs fail with “update check cancelled”
